core/translate/plan.rs

```diff
@@ -9,20 +9,34 @@ use sqlite3_parser::ast;
 use crate::{
     function::AggFunc,
     schema::{BTreeTable, Index},
-    util::normalize_ident,
     Result,
 };
 
+#[derive(Debug)]
+pub enum ResultSetColumn {
```

Comment:
i'll still try to get rid of this enum. we do need to encode whether a result set column contains aggregates (either directly eg `sum(id)` or indirectly `length(sum(id)`), and ideally, which